IMAGES.rst

```diff
@@ -0,0 +1,427 @@
+| ``ENTRYPOINT_FILE``               | Should point to entrypoint.sh     |
+|                                   | file in case of installation from |
+|                                   | the package or from GitHub URL.   |
+|                                   | See examples below                |
++-----------------------------------+-----------------------------------+
+| ``AIRFLOW_WWW``                   | In case of Airflow 2.0 it should  |
+|                                   | be "www", in case of Airflow 1.10 |
+|                                   | series it should be "www_rbac".   |
+|                                   | See examples below                |
++-----------------------------------+-----------------------------------+
+| ``AIRFLOW_SOURCES_FROM``          | Sources of Airflow. Should be set |
+|                                   | to "Dockerfile" to avoid costly   |
+|                                   | Docker context copying            |
+|                                   | in case of installation from      |
+|                                   | the package or from GitHub URL.   |
+|                                   | See examples below                |
++-----------------------------------+-----------------------------------+
+| ``AIRFLOW_SOURCES_TO``            | Target for Airflow sources. Set   |
+|                                   | to "/Dockerfile" to avoid costly  |
+|                                   | Docker context copying            |
+|                                   | in case of installation from      |
+|                                   | the package or from GitHub URL.   |
+|                                   | See examples below                |
++-----------------------------------+-----------------------------------+
+
+
+This builds production image in version 3.6 with default extras from the local sources:
+
+.. code-block::
+
+  docker build .
+
+This builds the production image in version 3.7 with default extras from 1.10.9 tag and
+requirements taken from v1-10-test branch in Github.
+Note that versions 1.10.9 and below have no requirements so requirements should be taken from head of
+the v1-10-test branch. Once we release 1.10.10 we can take them from the 1.10.10 tag. Also
+Note that in case of Airflow 1.10 we need to specify "www_rbac" instead of "wwww" for
+WWW_FOLDER argument.
+
+.. code-block::
+
+  docker build . \
+    --build-arg PYTHON_BASE_IMAGE="python:3.7-slim-buster" \
+    --build-arg PYTHON_MAJOR_MINOR_VERSION=3.7 \
+    --build-arg AIRFLOW_INSTALL_SOURCES="https://github.com/apache/airflow/archive/1.10.10.tar.gz#egg=apache-airflow" \
+    --build-arg CONSTRAINT_REQUIREMENTS="https://raw.githubusercontent.com/apache/airflow/1.10.10/requirements/requirements-python3.7.txt" \
+    --build-arg ENTRYPOINT_FILE="https://raw.githubusercontent.com/apache/airflow/1.10.10/entrypoint.sh" \
+    --build-arg SOURCES_FROM="Dockerfile" \
+    --build-arg SOURCES_TO="/Dockerfile" \
```

Comment:
```suggestion
    --build-arg AIRFLOW_SOURCES_FROM="Dockerfile" \
    --build-arg AIRFLOW_SOURCES_TO="/Dockerfile" \
```

Reply:
Feel free to create a PR for this fixes :)

Reply:
Yeah. I am back. Happy to review those :)

Reply:
Fixed in new PR coming.